paint-the-room/readme.md

```diff
@@ -0,0 +1,93 @@
+# Paint The Room 🧑‍🎨
+
+Estimating how long a task will take is a part of every job. In software development estimating how long a large task will take is difficult due to hidden complexity.
+
+Today we will be learning some effective techniques that allow us to still estimate tasks and plan through an interactive game.
+
+## Learning Objective 🧐
```

Comment:
Hi Caroline, please format these as learning objectives. Here is a worksheet to help you do this

https://docs.google.com/document/d/1NjeGQHCUFTplbj4tCA_lb-I9nugKRVh7PhcFwKXd-uo

Reply:
Very helpful link thank you @SallyMcGrath will revise the objects 👍